src/Type/BaseStatusType.php

```diff
@@ -34,16 +34,34 @@ abstract class BaseStatusType extends AbstractType
      */
     protected $name;
 
+    /**
+     * @deprecated Since 0.x, to be remove in 1.0. Use configureOptions instead.
+     *
+     * @var bool
+     */
+    protected $flip;
+
     /**
      * @param string $class
      * @param string $getter
      * @param string $name
+     * @param bool   $flip   reverse key/value to match sf2.8 and sf3.0 change
      */
-    public function __construct($class, $getter, $name)
+    public function __construct($class, $getter, $name, $flip = null)
     {
+        if (null === $flip) {
+            $flip = true;
+        } else {
+            @trigger_error(
+                'Calling '.__CLASS__.' class with a flip parameter is deprecated since 0.x, to be removed with 1.0',
+                E_USER_DEPRECATED
+            );
+        }
```

Comment:
Let's avoid both the `else` and the extra assignment
```suggestion
        if (null !== $flip) {
            @trigger_error(
                'Calling '.__CLASS__.' class with a flip parameter is deprecated since 0.x, to be removed with 1.0',
                E_USER_DEPRECATED
            );
        }
```